### ISSUE TITLE
Can't save recorded gpx anymore

### ISSUE THREAD

**Issue by hvdwolf** [osmandapp/Osmand#2967]
OsmAnd 2.4.3 beta, August 11 2016

When cycling recreationally  I always track my route. The last 2 days I can no longer save the gpx. When clicking "save current gpx track", the led stays red "saving" the track.
When I finally stop/start OsmAnd or even reboot the phone, the current track is still not saved and also now I can't save it either.

**Comment by vshcherb** [comment]
Unfortunately can't reproduce it. Potentially you need to wait a bit longer until it saves, if the save is not successful, after reboot it tries to do the same.

**Comment by vshcherb** [comment]
P.S. don't recall any latest changes in GPX save procedure

**Comment by hvdwolf** [comment]
Maybe I'm the only one. But I waited more than 10 minutes (didn't mention that) and I did reboot (I mentioned that).
I tried again. After 18 minutes it was still not saved. After 30 minutes it was saved. So it took somewhere between 18 and 30 minutes to save it.

**Comment by vshcherb** [comment]
This is very-very long, if you could say how many points where there and how big was the file in the end, it could help us to find the test data like that

**Comment by hvdwolf** [comment]
The track is 1 hour 16 mins cycling with a point every 5 seconds. I have much bigger track files for much longer cycling tours.
It is very, very strange.
every night I switch off my phone. This morning I switched it on. I could immediately access my track and display it. Upon tapping "back" OsmAnd kept "loading" and I could no longer access my tracks (+10 minutes). The rotating circle in top keeps rotating, but I can access my favorites. I closed the app completely, removed that last recorded track (you never know if it is somehow faulty) and restarted OsmAnd: still no access.
Then I rebooted my phone but after reboot still no access.
Finally I switched off my phone completely and switched it on again: I still can't access my tracks.
I will check my sdcard for corruption tonight.
